build_overrides/vulkan_validation_layers.gni

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2019 The Aquarium Project Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+#
+
+vulkan_headers_dir = "//third_party/vulkan-headers"
+vvl_spirv_tools_dir = "//third_party/SPIRV-Tools"
+vvl_glslang_dir = "//third_party/glslang"
+
+# Subdirectories for generated files
+vulkan_data_subdir = "vulkandata"
+vulkan_gen_subdir = ""
+
+dawn_vulkan_validation_layers_dir = "//third_party/vulkan-validation-layers"
+dawn_enable_vulkan_validation_layers = is_win&&is_debug
```

Comment:
Override `dawn_vulkan_validation_layers_dir` and `dawn_enable_vulkan_validation_layers` in build_overrides/dawn.gni, not build_overrides/vulkan_validation_layers.gni.

Reply:
Done